torch/testing/_internal/common_methods_invocations.py

```diff
@@ -635,6 +635,21 @@ def sample_inputs_gather(op_info, device, dtype, requires_grad):
                         0, torch.tensor(0, dtype=torch.int64, device=device))),
             )
 
+def sample_inputs_amax_amin(op_info, device, dtype, requires_grad):
+    test_cases = (
+        ((S, S, S), ()),
+        ((S, S, S), (1,)),
+        ((S, S, S), ((1, 2,),)),
+        ((S, S, S), (1, True,)),
+        ((), (0,)),
```

Comment:
Can you also add the scalar/no_args version as follows:
``((), tuple()),``

Reply:
Oops, I had missed that one! Thanks again for catching it!